listenbrainz/webserver/templates/profile/info.html

```diff
@@ -12,6 +12,15 @@ <h3> Delete your account </h3>
       <a href="{{ url_for("profile.delete") }}" style="color: white;"><span class="btn btn-info btn-lg" style="width: 300px">Delete your account</span></a>
     </p>
 
+    <h3> Delete your listens </h3>
+    <p>
+      This will delete all your listens imported into Listenbrainz.
```

Comment:
I think this text makes it sound like if you press the button your listens are gone, when in fact you'll be taken to another page. I personally don't think we need the extra page just yet -- all the other actions can be taken directly from the profile page. I think for now it would be better to add the delete listens function to the profile page.

Reply:
Okay.

Reply:
@mayhem Done :)